docs/pages/careers.tsx

```diff
@@ -388,12 +416,13 @@ function CareersContent() {
       <Container sx={{ py: { xs: 4, md: 8 } }}>
         <div>
           <Typography variant="h2" sx={{ my: 1 }} id="open-roles">
-            Open roles
+            Open roles (8)
           </Typography>
           <Typography color="text.secondary" sx={{ mb: 2, maxWidth: 450 }}>
```

Comment:
Should we increase the `maxWidth`? Maybe to match with the one for the description of the roles?

![image](https://user-images.githubusercontent.com/4512430/146748783-8679aaaa-9be5-4306-819c-fe7c01c72952.png)

Reply:
Sure, increased it a bit